src/docs/metrics/binance-derivatives/busd-binance-perpetual-open-interest-value/index.md

```diff
@@ -52,7 +52,7 @@ assets](<https://api.santiment.net/graphiql?variables=&query=%7B%0A%20%20getMetr
 
 ### SanAPI - Open Interest
 
-Available under the `busd_binance_open_interest` name.
+Available under the `busd_binance_open_interest` and `busd_bnb_open_interest` names.
```

Comment:
This is implemented like this:
```json
  {
    "human_readable_name": "Binance BUSD Exchange Open Value",
    "name": "busd_bnb_open_value",
    "metric": "busd_bnb_open_value",
    "aliases": [
      "busd_binance_open_value"
    ]
  ...
}
```

We need to decide which one is the preferred one and mention that the other name is an alias to this one, but is "deprecated" and should not be used. And people need to migrate.

Reply:
Fixed